{"name": "Получение задачи с невалидным ID", "status": "passed", "steps": [{"name": "Генерируем данные для новой задачи", "status": "passed", "attachments": [{"name": "Сгенерированные данные", "source": "15361643-c6b9-4b76-8555-864f0b7d6ee3-attachment.json", "type": "application/json"}], "start": 1746535622079, "stop": 1746535622080}, {"name": "Создаем задачу через API", "status": "passed", "attachments": [{"name": "Тело запроса", "source": "42a4081e-5e92-4413-aa69-653df0910630-attachment.json", "type": "application/json"}, {"name": "Ответ сервера", "source": "00b8f330-5d37-48c1-bca8-2cfee9971a85-attachment.json", "type": "application/json"}], "start": 1746535622080, "stop": 1746535622890}, {"name": "Отправляем GET запрос с некорректным ID: 999999999", "status": "passed", "start": 1746535622890, "stop": 1746535624621}, {"name": "Проверяем код ответа и сообщение об ошибке", "status": "passed", "start": 1746535624621, "stop": 1746535624621}], "parameters": [{"name": "wrong_id", "value": "999999999"}], "start": 1746535622079, "stop": 1746535624622, "uuid": "4d0d0a8c-ad9c-44de-9876-e31727dc0837", "historyId": "09f06eb74780ba176e64dcf477fc3e22", "testCaseId": "3e916150ead964e7afe384aca8020f86", "fullName": "tests.api.test_tasks.TestTasks#test_get_task_invalid", "labels": [{"name": "parentSuite", "value": "tests.api"}, {"name": "suite", "value": "test_tasks"}, {"name": "subSuite", "value": "TestTasks"}, {"name": "host", "value": "Egors-MacBook-Air.local"}, {"name": "thread", "value": "21580-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api.test_tasks"}]}